commands/send.js

```diff
@@ -0,0 +1,16 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { options } = require('request');
+const PubSub = require('pubsub-js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('send')
+        .setDescription('Send a message to the minecraft server you are connected to!')
+        .addStringOption(option => option.setName('message').setDescription('send message bitch').setRequired(true)),
+    async execute(interaction) {
+        await interaction.reply("Sent Message!").then(() => {
+            console.log("Actually did the thing (send)");
```

Comment:
i understand this is in progress, please remove such console logs before merging